api/repositories/filter/route_apps_filtering_test.go

```diff
@@ -0,0 +1,40 @@
+package filter_test
+
+import (
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"code.cloudfoundry.org/korifi/api/repositories/filter"
+)
+
+var _ = Describe("RoutesByAppGUID", func() {
+	var (
+		appGUID1     string
```

Comment:
`appGUID1` is never initialised, i.e. its default value `""` is being used. It should be set to a random uuid in `BeforeEach`